### PR TITLE
Odyssey Completion

### PR DESCRIPTION
**Pull request by Gnomeski**
### Description

- Resolves #200 

- The Origin Story of Meta Genesis

### Notes

- Confirm that it displays properly in the live preview before merging

- The following is an image preview on the MetaGame wiki

image

### CODE REVIEW

**Review by luxumbra — APPROVED** [review]
Checked locally and all good. :pray:  Congrats on your first PR @Gnomeski :partying_face:

**Comment by luxumbra** [comment]
@hammadj could you approve the deploy preview please when you get a chance. I've checked this locally and all good.

**Comment by peth-yursick** [comment]
What's the status of this? Still waiting for @hammadj to approve in Vercel?

**Comment by luxumbra** [comment]
> What's the status of this? Still waiting for @hammadj to approve in Vercel?

Yeah. It needs to deploy the preview build as a final check before it can be merged.

File: docs/wtf-is-metagame/origin-story.md
Comment (hammadj) [inline docs/wtf-is-metagame/origin-story.md:5]: I think you can remove this bit since it already shows up in the title above. Also, wondering if we should mention this page is about the "MetaGame Origin", some people might not understand what "The MetaGenesis Odyssey" means or what the purpose of this page in the docs is.

**Review by peth-yursick — APPROVED** [review]

**Pull request merged**
